bundler/spec/dependabot/bundler/update_checker/version_resolver_spec.rb

```diff
@@ -142,11 +141,6 @@
 
           let(:dependency_files) { bundler_project_dependency_files("bundler_specified") }
 
-          before do
-            stub_request(:get, rubygems_url + "versions/bundler.json").
-              to_return(status: 200, body: fixture("rubygems_responses", "versions-bundler.json"))
-          end
-
```

Comment:
This code was meant to never actually make it to the repository, it seems just an oversight that it landed. See https://github.com/dependabot/dependabot-core/pull/3887#discussion_r651000907.